pages/[id].jsx

```diff
@@ -1,26 +1,29 @@
 import Head from 'next/head';
 import SkipToContent from '../components/SkipToContent';
 import apiService from '../services/apiService';
+import getFullName from '../util/getFullName';
 
 export default function DetailPage({ fetchedTeamMember }) {
+  const { firstName, lastName } = fetchedTeamMember;
+  const fullName = getFullName(firstName, lastName);
```

Comment:
This is an opinion based comment so feel free to disagree but wouldn't it be nice we moved the logic for deconstructing the object containing the firstName and lastName data into the `getFullName` function.  That way we can write a little less code each time we need to call the function. Here is would look like 
```
const fullName = getFullName(fetchedTeamMember); 
``` 
instead of 
```
const { firstName, lastName } = fetchedTeamMember;
const fullName = getFullName(firstName, lastName); 
```

Reply:
I'm convinced!